makefiles/S3.toml

```diff
@@ -39,6 +39,7 @@ script = '''
 mkdir -p ~/.fluvio/extensions
 ./target/${TARGET}/release/fluvio install fluvio/fluvio-package;
 ./target/${TARGET}/release/fluvio package publish ./target/${TARGET}/release/fluvio --version=${FLUVIO_VERSION};
+./target/${TARGET}/release/fluvio package bump latest;
```

Comment:
There's a small chance of a race condition between the macos release build and the ubuntu release build. Ideally, this would be done after all of them are complete. Doing this correctly in CI is annoying because you gotta re-install fluvio :/. Maybe we wait for #579 and #563?

Reply:
sure